TypeCobol/Compiler/CompilationUnit.cs

```diff
@@ -86,9 +89,6 @@ public void RefreshCodeElementsDocumentSnapshot()
                 // Start perf measurement
                 var perfStatsForParserInvocation = PerfStatsForCodeElementsParser.OnStartRefreshParsingStep();
```

Comment:
This must be called at the beginning of the method.

Reply:
changed